internal/hscan/hscan.go

```diff
@@ -107,28 +107,38 @@ func decodeBool(f reflect.Value, s string) error {
 }
 
 func decodeInt(f reflect.Value, s string) error {
-	v, err := strconv.ParseInt(s, 10, 0)
+	v, err := strconv.ParseInt(s, 10, 64)
 	if err != nil {
 		return err
 	}
+	if f.OverflowInt(v) {
```

Comment:
This is tiny bit slower than passing `bitSize` to `ParseInt`. And `OverflowInt` just calculates the `bitSize` for us and does a check - https://github.com/golang/go/blob/master/src/reflect/value.go#L1413